### PR TITLE
fix(call): Improv user ui in call

### LINKED ISSUE #1493: bug(ui): Members render off Call UI when too many members
**Issue:**
When you have 10+ users in the group call, the members will start to render off the call UI 

**Screenshot:**
Screenshot 2023-11-09 at 12 17 24 PM

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Fixes the problem where having a lot of users in a call makes it render off screen

### Which issue(s) this PR fixes 🔨

- Resolve #1493

### CODE REVIEW

File: kit/src/components/context_menu/context.js [hunk 1]
```diff
@@ -1,37 +1,53 @@
+let click = (toggle) => function (ev) {
```
Comment (sdwoodbury) [inline kit/src/components/context_menu/context.js:1]: i'm not familiar with this syntax. based off of [this](https://www.w3schools.com/Js/js_es6.asp#mark_arrow) it looks like the arrow can be used to make a lambda like this `(..) => {}` but I'm not familiar with `(..)  => function(..) {}`. is this valid javascript? where can I read more about it?

File: kit/src/components/context_menu/context.js [hunk 2]
```diff
@@ -1,37 +1,53 @@
...
+  const menus = document.getElementsByClassName("context-menu")
+  // Select the current one
+  const context_menu = document.getElementById("UUID-context-menu")
+  let hidden = context_menu.classList.contains("hidden");
+  for (var i = 0; i < menus.length; i++) {
+    menus.item(i).classList.add("hidden")
+  }
+  if (toggle && !hidden) {
+    context_menu.classList.add("hidden")
+    return
```
Comment (sdwoodbury) [inline kit/src/components/context_menu/context.js:14]: this should probably have a boolean return type.

File: ui/src/components/media/calling.rs [hunk 3]
```diff
@@ -681,15 +677,82 @@ pub struct CallUserImageProps {
...
+                    }
+                    Err(e) => {
+                        log::error!("eval receiver failed: {:?}", e);
+                    }
+                }
+            }
+        }
+    });
+    let visible_amount = *amount.get() as usize;
+    let (visible, context) = if visible_amount > cx.props.participants.len() {
```
Comment (sdwoodbury) [inline ui/src/components/media/calling.rs]: this probably needs to be `visible_amount >= cx.props.participants.len()`

File: kit/src/components/context_menu/context.js (hunk 1 above)
Comment (Flemmli97) [reply kit/src/components/context_menu/context.js:1]: its just a function creating a function.
so calling click(true) returns the function with toggle being true

File: kit/src/components/context_menu/context.js (hunk 2 above)
Comment (Flemmli97) [reply kit/src/components/context_menu/context.js:14]: actually the end return is not needed instead so removed that

**Review by sdwoodbury — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**